scripts/zones/Halvung/mobs/Gurfurlur_the_Menacing.lua

```diff
@@ -1,46 +1,53 @@
 -----------------------------------
 -- Area: Halvung
---  MOB: Gurfurlur the Menacing
+-- MOB: Gurfurlur the Menacing
+-- !pos -59.000 -23.000 3.000 62
 -----------------------------------
 require("scripts/globals/titles");
+require("scripts/globals/status");
+require("scripts/zones/Halvung/MobIDs");
+mixins = {require("scripts/mixins/job_special")};
 -----------------------------------
 
-function onMobSpawn(mob)
-end;
-
 function onMobEngaged(mob,target)
-
-    local gurfurlur = mob:getID()
-    SpawnMob(gurfurlur+1):updateEnmity(target);
-    SpawnMob(gurfurlur+2):updateEnmity(target);
-    SpawnMob(gurfurlur+3):updateEnmity(target);
-    SpawnMob(gurfurlur+4):updateEnmity(target);
-
+    for i = GURFURLUR_THE_MENACING + 1, GURFURLUR_THE_MENACING + 4 do
+        SpawnMob(i):updateEnmity(target);
+    end
 end;
 
 function onMobFight(mob,target)
+    if (mob:getBattleTime() % 15 < 2 and mob:getBattleTime() > 10) then
```

Comment:
im not sure what this is supposed to do

Reply:
its a Modulo operation thats does a check for being true every 15 seconds to respawn add if its depoped. but looks like i had wrong information it should be % 60 for every 1 min